screens/Menu/ShuttleBus.js

```diff
@@ -66,7 +283,87 @@ export const styles = StyleSheet.create({
     menuButtonContainer: {
```

Comment:
Change this to 
`   menuButtonContainer: {
        width: "100%",
        height: "15%",
        top: "17%"
    },`

Because we cannot see the the sideMenuIcon